gobblin-core/src/main/java/gobblin/filesystem/InstrumentedHDFSFileSystem.java

```diff
@@ -40,8 +53,67 @@
 public class InstrumentedHDFSFileSystem extends DistributedFileSystem {
 
   public static final String INSTRUMENTED_HDFS_SCHEME = "instrumented-hdfs";
-
+  public static final String HDFS_METRIC_CONTEXT_NAME = "hdfsMetricContext";
   private static final String HDFS_SCHEME = "hdfs";
+  private MetricContext metricContext;
+
+  protected final Closer closer;
+
+  // Below are HDFS metrics
+  @VisibleForTesting
+  protected final Timer listStatusPathTimer;
```

Comment:
There are a few listStatus may duplicate the timer count. 
listStatus(path[]) -> listStatus(path[], filter) ---> listStatus(path)
listStatus(path, filter) ________________________/

If we only care about the final operation listStatus(path), then only listStatusPathTimer should be inspected.